proto/protocolapis/astria/protocol/transactions/v1alpha1/types.proto

```diff
@@ -181,3 +182,22 @@ message BridgeLockAction {
   // will receive the bridged funds
   string destination_chain_address = 5;
 }
+
+message FeeChangeAction {
+  // note that the proto number ranges are doubled from that of `Action`.
+  // this to accomodate both `base_fee` and `byte_cost_multiplier` for each action.
+  oneof value {
+    // core protocol fees are defined on 1-20
+    bytes transfer_base_fee = 1;
```

Comment:
Should these by bytes or typed as u128 or uint64?

It might be cleaner to have them all be uint128 and then because it's a message they are optional. could get rid of the `oneof`, can update as many as you want at once anything not set stays the same?

Reply:
updated to make each `oneof value` type `uint128` - i think it's better to leave it as a oneof for now, as changing multiple fees in one action can get more complex than just bundling a tx with multiple `FeeChangeAction`s in it.